lib/gui/app/pages/main/Flash.tsx

```diff
@@ -299,7 +306,11 @@ export const Flash = ({ shouldFlashStepBeDisabled, goToSuccess }: any) => {
 					done={() => handleFlashErrorResponse(true)}
 					action={'Retry'}
 				>
-					<Txt>{errorMessage}</Txt>
+					<Txt>
+						{_.map(errorMessage.split('\n'), (message, key) => (
+							<p key={`${message}-${key}`}>{message}</p>
```

Comment:
`key={key}` should be enough.

Reply:
It gave me warnings using just one or the other, but admit it was in the early stages and things have changed. I'll try that